cmd/daserver/daserver.go

```diff
@@ -85,12 +110,18 @@ func startup() error {
 		return err
 	}
 	var dasImpl das.DataAvailabilityService
-	if mode == das.LocalDataAvailability {
-		dasImpl, err = das.NewLocalDiskDataAvailabilityService(serverConfig.DAConf.LocalDiskDataDir, uint64(serverConfig.DAConf.SignerMask))
+	switch mode {
```

Comment:
It's a little different now that I made the one in `node.go` exhaustive since it can use the OnchainDataAvailability mode but `daserver.go` can't. I'll wait and see on this since we are adding some more modes like S3, redis, etc and different combinations may be needed on the node vs on the server.